src/_includes/layouts/base.html

```diff
@@ -52,12 +52,35 @@
     <meta name="twitter:image" content="{{ cover_image }}">
     <meta name="twitter:creator" content="@geteslint">
 
-
-    <script type="module">
+    <script>
         // This is a capable browser, let's improve the UI further!
         document.documentElement.classList.add("enhanced");
         document.documentElement.classList.remove('no-js');
     </script>
+ 
+    <style>
```

Comment:
Can this be moved into a .scss file?

Also, what if we made a `js-only` class that we can apply to all the buttons that should be hidden when there JS is disabled? That way, we wouldn't need to list out each element individually.

Reply:
Sure, I will try it.

Reply:
yes, changes are done.